GsnDevProvider.ts

```diff
@@ -0,0 +1 @@
+export * from './src/relayclient/GsnDevProvider'
```

Comment:
wait, all these files are in the project root? Why?

Reply:
I know. didn't want them there.
But I wanted the usage string to be the simplest
`require( 'opengsn/gsn/dist/GsnDevProvider' )`
(and actually I wanted to get rid of "dist" too...)

do you know another way to make it?
I can copy them into the root of the package during "pack", but then the paths must be broken in the git checked in version, so this seemed like a fair tradeoff..

Reply:
removed those file (by generating them in the `prepublish`